content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -0,0 +1,160 @@
+  ```
+
+#### Additional information about kubeadm join
+
+{{< note >}}
+To specify an IPv6 tuple for `<control-plane-host>:<control-plane-port>`, IPv6 address must be enclosed in square brackets, for example: `[2001:db8::101]:2073`.
+{{< /note >}}
+
+If you do not have the token, you can get it by running the following command on the control plane node:
+
+```bash
+kubeadm token list
+```
+
+The output is similar to this:
+
+```console
+TOKEN                    TTL  EXPIRES              USAGES           DESCRIPTION            EXTRA GROUPS
+8ewj1p.9r9hcjoqgajrj4gi  23h  2018-06-12T02:51:28Z authentication,  The default bootstrap  system:
+                                                   signing          token generated by     bootstrappers:
+                                                                    'kubeadm init'.        kubeadm:
+                                                                                           default-node-token
+```
+
+By default, node join tokens expire after 24 hours. If you are joining a node to the cluster after the
+current token has expired, you can create a new token by running the following command on the
+control plane node:
+
+```bash
+kubeadm token create
+```
+
+The output is similar to this:
+
+```console
+5didvk.d09sbcov8ph2amjw
+```
+
+If you don't have the value of `--discovery-token-ca-cert-hash`, you can get it by running the
+following commands on the control plane node:
+
+```bash
+sudo cat /etc/kubernetes/pki/ca.crt | openssl x509 -pubkey  | openssl rsa -pubin -outform der 2>/dev/null | \
+   openssl dgst -sha256 -hex | sed 's/^.* //'
+```
+
+The output is similar to:
+
+```console
+8cb2de97839780a412b93877f8507ad6c94f73add17d5d7058e91741c9d5ec78
+```
+
+The output of the `kubeadm join` command should look something like:
+
+```
+[preflight] Running pre-flight checks
+
+... (log output of join workflow) ...
+
+Node join complete:
+* Certificate signing request sent to control-plane and response
+  received.
+* Kubelet informed of new secure connection details.
+
+Run 'kubectl get nodes' on control-plane to see this machine join.
+```
+
+A few seconds later, you should notice this node in the output from `kubectl get nodes`
+when run on the control plane node.
+
+### Network configuration
+
+CNI setup on clusters mixed with Linux and Windows nodes requires more steps than just
+running `kubectl apply` on a manifest file. Additionally, the CNI plugin running on control
+plane nodes must be prepared to support the CNI plugin running on Windows worker nodes.
+
+{{% thirdparty-content %}}
+
+Only a few CNI plugins currently support Windows. Below you can find individual setup instructions for them:
+* [Flannel](https://sigs.k8s.io/sig-windows-tools/guides/flannel.md)
+* [Calico](https://sigs.k8s.io/sig-windows-tools/guides/calico.md)
+* [Calico Operator](https://sigs.k8s.io/sig-windows-tools/guides/calico_operator.md)
```

Comment:
I'd prefer to link to calico's official installation scripts and not including links to the sig-windows guides.  They have gone stale and community hasn't been able to maintain them

https://docs.tigera.io/calico/latest/getting-started/kubernetes/windows-calico/